sentry/src/main/java/io/sentry/SentryStackTraceFactory.java

```diff
@@ -39,7 +39,7 @@ public SentryStackTraceFactory(
 
           // we don't want to add our own frames
           final String className = item.getClassName();
-          if (className.startsWith("io.sentry.")) {
+          if (className.startsWith("io.sentry.") && !className.startsWith("io.sentry.samples.")) {
```

Comment:
not sure if this should be part of the SentryStackTraceFactory, another way of looking into it is to rename the sample packages, I see its a quick fix but seems the wrong approach, wdyt?

Reply:
Today I am fine with both. What would be the proper package name for samples?

Reply:
Let's go for this one instead of renaming everything, we have quite a few samples already.